test/functional-tests/pageObjects/ccd-components/fields/ccdMultiSelectField.js

```diff
@@ -0,0 +1,87 @@
+class CcdMultiSelectField {
+
+  constructor(css){
+      this.css = css;
+      this.labels = this._getLabels();
+  }
+
+  /**
+   * Will randomly select any multi select option
+   */
+  async selectAnyOneElement(){
+    let options = await this._getMultiSelectTextValues();
+    let elementListSize = await options.length;
+    let randomOptionArrayInt = await RandomUtils.generateRandomInt(1, await elementListSize);
+    let optionToSelect = await options[randomOptionArrayInt-1];
+    await optionToSelect.click();
+  }
+
+  /**
+   * Check if field is ready to type
```

Comment:
update doc, perhaps just '... is present and enabled' could do

Reply:
Done